app/assets/javascripts/discourse/app/templates/group/permissions.hbs

```diff
@@ -0,0 +1,20 @@
+<section class="user-content">
```

Comment:
`user-content` is a weird class name for something related to a group

Reply:
This is just the class all the group views use for the section

https://github.com/discourse/discourse/blob/1ca81fbb95ec32fce448c477cbb824e884574c93/app/assets/javascripts/discourse/app/templates/group/messages.hbs#L15

https://github.com/discourse/discourse/blob/1ca81fbb95ec32fce448c477cbb824e884574c93/app/assets/javascripts/discourse/app/templates/group/manage.hbs#L12

https://github.com/discourse/discourse/blob/1ca81fbb95ec32fce448c477cbb824e884574c93/app/assets/javascripts/discourse/app/templates/group/activity.hbs#L13